tests/connect_redirect/connect_redirect_tests.cpp

```diff
@@ -361,10 +349,9 @@ _update_policy_map(
 
 void
 connect_redirect_test(
-    _In_ const struct bpf_object* object,
     _In_ client_socket_t* sender_socket,
     _In_ sockaddr_storage& destination,
```

Comment:
To your point, shouldn't this be Inout?
```suggestion
    _Inout_ sockaddr_storage& destination,
```